pkg/security/probe/consts.go

```diff
@@ -243,13 +243,13 @@ var (
 		"O_DIRECTORY": syscall.O_DIRECTORY,
 		"O_DSYNC":     syscall.O_DSYNC,
 		"O_FSYNC":     syscall.O_FSYNC,
-		"O_LARGEFILE": syscall.O_LARGEFILE,
-		"O_NDELAY":    syscall.O_NDELAY,
-		"O_NOATIME":   syscall.O_NOATIME,
-		"O_NOCTTY":    syscall.O_NOCTTY,
-		"O_NOFOLLOW":  syscall.O_NOFOLLOW,
-		"O_NONBLOCK":  syscall.O_NONBLOCK,
-		"O_RSYNC":     syscall.O_RSYNC,
+		//"O_LARGEFILE": syscall.O_LARGEFILE, golang defines this as 0
```

Comment:
Shouldn't you add the same comment for O_RDONLY?

Reply:
Or say something like "syscall.O_LARGEFILE and syscall.O_RDONLY both equal 0 - we are only interested in O_RDONLY", or something like that?

Reply:
I'll update it in following PR